libs/core/src/lib/pagination/pagination.component.html

```diff
@@ -4,7 +4,7 @@
             <ng-container [ngTemplateOutlet]="displayTextTemplate || showingItems" [ngTemplateOutletContext]="{ showing: currentShowing }"></ng-container>
         </span>
         <nav class="fd-pagination__nav" *ngIf="totalItems >= itemsPerPage">
-            <a
+            <div
```

Comment:
I don't think you can use a `div`
Please check the a11y guidelines for pagination. Here are some links:
https://www.a11ymatters.com/pattern/pagination/

https://a11y-style-guide.com/style-guide/section-navigation.html#kssref-navigation-pagination

Reply:
used aria-current instead of aria-selected for link tag.